### PR TITLE
Modify your 128 issue

### PR DESCRIPTION
**Pull request by udrs**
Thank you for your great job.

### CODE REVIEW

File: tools/whipinto/src/test.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        Ok(())
+    }
+
+    pub async fn print_local_description(&self) {
+        let current_local_description = self.internal.current_local_description.lock().await;
+        println!("Current: {:?}", *current_local_description);
+    }
+}
+
+#[tokio::main]
```
Comment (a-wing) [inline tools/whipinto/src/test.rs:43]: https://doc.rust-lang.org/book/ch11-01-writing-tests.html

**Review by a-wing — CHANGES_REQUESTED** [review]
Thank you very much your contribution

This commit fix #128 right?

The code need some a little change:
1. *your change upstream, you need change `Cargo.toml`
2. use `cargo fmt` format all code
3. use `cargo clippy --all-targets --all-features --workspace -- -D warnings` lint your code
4. *CLA sign

* * *
**1. your change upstream, you need change `Cargo.toml`**

your patch upstream: https://github.com/webrtc-rs/webrtc/commit/ae93e81a493c004b0270603d539ad9a40b7b3064. 

you need change `Cargo.toml`  in 

https://github.com/binbat/live777/blob/b9b397f30aa434a5b36266b9f9404beaa6549738/Cargo.toml#L21-L23

* * *

**4. CLA sign**

Please read this document and sign at the end

https://github.com/binbat/live777/blob/main/.github/CLA.md

You can reference this: https://github.com/binbat/live777/pull/95

File: liveman.toml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+[http]
```
[the remark this answers is not in the document]
Comment (udrs) [reply liveman.toml:1]: Issues mentioned are solved. Thank you for your great job and check.

File: tools/whepfrom/src/main.rs [hunk 3]
```diff
@@ -157,11 +157,55 @@ async fn webrtc_start(
...
+    //}
+
+
+
+    peer.set_configuration(current_config.clone()).await?;
+
+
+
+
+    // 获取更新后的配置
```
[the remark this answers is not in the document]
Comment (udrs) [reply tools/whepfrom/src/main.rs:187]: Issues mentioned are solved. Thank you for your great job and check.

File: tools/whepfrom/src/main.rs [hunk 4]
```diff
@@ -157,11 +157,55 @@ async fn webrtc_start(
...
+    let updated_config = peer.get_configuration().await;
```
[the remark this answers is not in the document]
Comment (udrs) [reply tools/whepfrom/src/main.rs:188]: Verify that the configuration was updated successfully

File: tools/whepfrom/src/main.rs [hunk 5]
```diff
@@ -157,11 +157,55 @@ async fn webrtc_start(
     let (answer, _ice_servers) = client
         .wish(peer.local_description().await.unwrap().sdp.clone())
         .await?;
+    
+    let mut current_config = peer.get_configuration().await;
+
+    //println!("Original ICE Servers:");
```
[the remark this answers is not in the document]
Comment (udrs) [reply tools/whepfrom/src/main.rs:163]: code Removed

File: tools/whipinto/src/test.rs (hunk 1 above)
Comment (udrs) [reply tools/whipinto/src/test.rs:43]: test.rs written

File: tools/whepfrom/src/main.rs [hunk 6]
```diff
@@ -157,11 +157,55 @@ async fn webrtc_start(
...
+    //    println!("Credential: {}", ice_server.credential);
+    //}
+
+
 
     peer.set_remote_description(answer)
         .await
         .map_err(|error| anyhow!(format!("{:?}: {}", error, error)))?;
-
+
```
[the remark this answers is not in the document]
Comment (udrs) [reply tools/whepfrom/src/main.rs]: space removed

File: tools/whepfrom/src/main.rs [hunk 7]
```diff
@@ -157,11 +157,55 @@ async fn webrtc_start(
     let (answer, _ice_servers) = client
```
[the remark this answers is not in the document]
Comment (udrs) [reply tools/whepfrom/src/main.rs]: name changed

**Review by udrs — COMMENTED** [review]
set getconfiguration function and corresponding test.rs file

**Review by a-wing — APPROVED** [review]

**Pull request merged**